configapp/configapp.py

```diff
@@ -109,10 +155,21 @@ def data_image(filename):
     return send_from_directory(directory, filename)
 
 
+def get_img_from_subdir(from_dir, img):
+    if '/' not in img:
+        pass
```

Comment:
@Adman if I understand this correctly, pass will make this function return `None`. What does that do server-wise? What status code will such a return value generate?

I suggest that in this case it should respond with the [400 HTTP status code](https://httpstatuses.com/400).